src/sbt-test/sbt-1.3/testkit/build.sbt

```diff
@@ -0,0 +1,38 @@
+import _root_.scalafix.sbt.{BuildInfo => Versions}
+
+inThisBuild(
+  List(
+    scalaVersion := Versions.scala212,
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision
+  )
+)
+
+val rules = project
+  .disablePlugins(ScalafixPlugin)
+  .settings(
+    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % Versions.scalafixVersion
+  )
+
+lazy val input = project
+
+lazy val output = project
+
+lazy val tests = project
+  .settings(
+    libraryDependencies +=
+      "ch.epfl.scala" % "scalafix-testkit" % Versions.scalafixVersion % Test cross CrossVersion.full,
+    scalafixTestkitOutputSourceDirectories :=
+      sourceDirectories.in(output, Compile).value,
+    scalafixTestkitInputSourceDirectories :=
+      sourceDirectories.in(input, Compile).value,
+    scalafixTestkitInputClasspath :=
+      fullClasspath.in(input, Compile).value,
+    scalafixTestkitInputScalacOptions :=
+      scalacOptions.in(input, Compile).value,
+    scalafixTestkitInputScalaVersion :=
+      scalaVersion.in(input, Compile).value,
+    mainClass in (Test, run) := Some("fix.IntputOutputSuite")
+  )
+  .dependsOn(input, rules)
+  .enablePlugins(ScalafixTestkitPlugin)
```

Comment:
I am not familiar with Docusaurus, but could we capture that somehow (like https://developer.lightbend.com/docs/paradox/current/directives/snippets.html) into https://github.com/scalacenter/scalafix/blob/master/docs/developers/setup.md#library-api? I guess it's tricky because it's not in the same repo...

Reply:
There's nothing like this out of the box that I'm aware of. It's possible to implement somehing like this on top of mdoc with custom modifiers https://scalameta.org/mdoc/docs/modifiers.html#stringmodifier